std/socket.d

```diff
@@ -3522,8 +3522,8 @@ class UdpSocket: Socket
             @property @trusted Address localAddress() { assert(0); }
             @trusted ptrdiff_t send(const(void)[] buf, SocketFlags flags) { assert(0); }
             @safe ptrdiff_t send(const(void)[] buf) { assert(0); }
-            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, Address to) { assert(0); }
-            @safe ptrdiff_t sendTo(const(void)[] buf, Address to) { assert(0); }
+            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, in Address to) { assert(0); }
+            @safe ptrdiff_t sendTo(const(void)[] buf, in Address to) { assert(0); }
```

Comment:
Isn't that issue about `const` methods?
But you are right,  this PR would break the existing overrides.

Thoughts on how we could resolve this?
Its pretty prolific having to remove const all the way through my stack because one method was missing it :/.